helpers/path.go

```diff
@@ -174,9 +174,17 @@ func AbsPathify(inPath string) string {
 	return filepath.Clean(filepath.Join(viper.GetString("WorkingDir"), inPath))
 }
 
+func GetStaticDirPath() string {
+	return AbsPathify(viper.GetString("StaticDir"))
+}
+
+func GetThemesDirPath() string {
+	return AbsPathify("themes"+string(os.PathSeparator)+viper.GetString("theme")) + string(os.PathSeparator) + "static" + string(os.PathSeparator)
```

Comment:
How about using `filepath.Join` (http://golang.org/pkg/path/filepath/#Join) instead of using raw `os.PathSeparator`? I think it makes the line more readable